src/main/java/com/adyen/serializer/DateTimeGMTSerializer.java

```diff
@@ -23,26 +23,22 @@
 import java.lang.reflect.Type;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.TimeZone;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonPrimitive;
 import com.google.gson.JsonSerializationContext;
 import com.google.gson.JsonSerializer;
 
 public class DateTimeGMTSerializer implements JsonSerializer<Date> {
-
-    // YYYY-MM-DDTHH:mm:ss.sssZ
-    public static final String DATE_FORMAT = "YYYY-MM-DD'T'HH:mm:ss.sss'Z'";
+    public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
 
     /**
      * Format to ISO 8601 format: YYYY-MM-DDTHH:mm:ss.sssZ
-     *
-     * @param date
-     * @param typeOfSrc
-     * @param context
-     * @return
      */
+    @Override
     public JsonElement serialize(Date date, Type typeOfSrc, JsonSerializationContext context) {
         SimpleDateFormat formatter = new SimpleDateFormat(DATE_FORMAT);
+        formatter.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Is it possible to use the timezone of the merchant ?

Reply:
since we add the Z in the end, we need to be in UTC
https://en.wikipedia.org/wiki/ISO_8601#UTC